react-app/src/Finish/index.js

```diff
@@ -40,14 +39,14 @@ export default function Finish({ db }) {
       <SafeAreaView style={styles.lowerWrapper}>
         <Header />
         <Image />
-        <Buttons />
         <Lottie
           source={require('../../assets/animation/62717-confetti.json')}
           autoPlay
           loop
           style={styles.animation}
           resizeMode='cover'
         />
+        <Buttons />
```

Comment:
it seems like just moving the '<Buttons />' to here.
May I ask you why those buttons didn't correctly work before?

Reply:
Generally the components which have `position:absolute` attribute and render later will cover the ones render earlier. The animation component was set to the size almost cover whole screen, so it covered the buttons previously.